src/utils/config.ts

```diff
@@ -1,11 +1,33 @@
 import { makeConfig } from 'cleaner-config'
-import { asNumber, asObject, asOptional, asString } from 'cleaners'
+import {
```

Comment:
If you recall us talking about adding constants from the environment to allow changing config at run time?

Here is an example of it looks like, it's from @LPeterson18's fee-tracker: 
https://github.com/EdgeApp/edge-fee-tracker/blob/aec7a98cd043af08537ab3aedaa22391a1c40653/src/utils/config.ts#L4